scrapy/crawler.py

```diff
@@ -123,17 +161,43 @@ def _setup_crawler_logging(self, crawler):
             crawler.signals.connect(log_observer.stop, signals.engine_stopped)
 
     def stop(self):
+        """
+        Stops simultaneously all the crawling jobs taking place.
+
+        Returns a deferred that is fired when they all have ended.
+        """
         return defer.DeferredList([c.stop() for c in list(self.crawlers)])
 
     @defer.inlineCallbacks
     def join(self):
-        """Wait for all managed crawlers to complete"""
+        """
+        join()
```

Comment:
Listing `:members:` explicitly sounds good to me - it allows to add docstrings to methods we don't want to promote, and these classes don't have lots of methods, so it is not a lot of typing. IMHO the less tricks the better :)